src/main/webapp/assets/js/feedScript.js

```diff
@@ -644,28 +658,47 @@ async function addPosts(posts) {
     postCard.setAttribute('id', post.id);
     postCard.setAttribute('tabindex', '0');
 
+    // Adds the image from the image serving URL.
+    // If there is not a valid URL, then display the stock SVG.
+    const cardImage = document.createElement('img');
+    cardImage.setAttribute('class', 'card-image');
+    let imageSource = post.imageServingUrl;
+    if (imageSource === 'no image') {
```

Comment:
why use `'no image'` instead of `null` to indicate that there isn't an image?

Reply:
+1. Use null

Reply:
Changed!